app/javascript/packs/pages/VolunteerConfirmation.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import VolunteerAndDoctor from "../components/illustrations/VolunteerAndDoctor";
+import BoldLink from "../components/BoldLink";
+import SocialMediaGrid from "../components/SocialMediaGrid";
+import SocialDistanceNotice from "../components/SocialDistanceNotice";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    alignItems: "center",
+    paddingTop: theme.spacing(5),
+    paddingBottom: theme.spacing(10),
+    flexDirection: "column",
```

Comment:
Not necessary to change but as a heads up, `<Box />` accepts flex css properties as props